spacy/structs.pxd

```diff
@@ -72,3 +76,32 @@ cdef struct TokenC:
     attr_t ent_type # TODO: Is there a better way to do this? Multiple sources of truth..
     attr_t ent_kb_id
     hash_t ent_id
+
+
+# Internal struct, for storage and disambiguation of entities.
+cdef struct EntryC:
```

Comment:
If this will live in the `structs.pxd`, then unfortunately we need a less vague name than `EntryC`. `KBEntryC` is ugly, but I can't immediately do better.

Reply:
I don't mind either way - I can also move it back to kb.pxd?

Reply:
Renamed to `KBEntryC` for now